suitcase/pizza-box/handlers.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+import humanize
+import pandas as pd
+import os
+from databroker.tests.utils import temp_config
```

Comment:
Please remove the databroker imports entirely. Since databroker imports suitcase, we should avoid making suitcase import databroker. It's OK to import it in some _tests_ (which might be the intent here?) but not in the primary code.

Reply:
Yes, please remove `uuid`, `humanize`, `databroker` from here @elParcero . I think you left these over from your test script :-)